### ISSUE TITLE
spring boot3.3 integration redisson boot with a lot of udp ports

### ISSUE THREAD

**Issue by xupy521** [redisson/redisson#6163]
![image](https://github.com/user-attachments/assets/8e23b0aa-1db0-45b4-b079-53caf6152c6d)

How to cancel boot on these udp ports？

**Comment by lyrric** [comment]
which redisson version?

**Comment by mrniko** [comment]
These are DNS resolver connections. You can disable them by setting dnsMonitoringInterval = -1

**Comment by xupy521** [comment]
> These are DNS resolver connections. You can disable them by setting dnsMonitoringInterval = -1

![image](https://github.com/user-attachments/assets/1535a8b4-eeb2-4876-b6eb-ac6c49cb076c)
![image](https://github.com/user-attachments/assets/869b6453-f184-407b-9651-2b0fc68b9558)

I set dnsMonitoringInterval = -1, but this problem still occurs. The redisson version remains the same. spring boot version 3.2.2 does not have this problem. The upgrade to 3.2.5 or later has this problem

**Comment by xupy521** [comment]
> These are DNS resolver connections. You can disable them by setting dnsMonitoringInterval = -1

![image](https://github.com/user-attachments/assets/596041a1-fafc-4fa6-8681-f6ddbc1fd097)
I found that the version of Netty-resolver that spring boot3.3.3 relies on is 4.1.112.Final, and I reduced the version of this dependency to 4.1.105.Final to solve this problem. Is the difficulty a problem of netty version? What if I want to upgrade netty-resolver to 4.1.112.Final?
